packages/clay-sticker/src/ClaySticker.js

```diff
@@ -30,13 +30,10 @@ ClaySticker.STATE = {
 	 * Render ClayIcon in the ClaySticker element.
 	 * @instance
 	 * @memberof ClaySticker
-	 * @type {?Object|undefined}
+	 * @type {?string|undefined}
 	 * @default undefined
 	 */
-	icon: Config.shapeOf({
```

Comment:
Hey @carloslancha, are we separating the `icon` to make it easier to use in JSP or to make it clearer?

Reply:
Both, we'll probably need to change some more API with that purpose in the next episode